docs/pages/build-reference/e2e-tests.md

```diff
@@ -243,6 +243,115 @@ If you have set up everything correctly you should see the successful test resul
 
 <img src="/static/images/eas-build/tests/03-logs.png" style={{maxWidth: "100%" }} />
 
+### 6. Upload screenshots of failed test cases
+
+This step is optional but highly recommended. When an E2E test case fails, it can be helpful to see the screenshot of the application state. EAS Build makes it easy to upload any arbitrary build artifacts using the [`buildArtifactPaths`](/build-reference/eas-json.md#buildartifactpaths) field in **eas.json**.
```

Comment:
```suggestion
> This step is optional but highly recommended. 

When an E2E test case fails, it can be helpful to see the screenshot of the application state. EAS Build makes it easy to upload any arbitrary build artifacts using the [`buildArtifactPaths`](/build-reference/eas-json.md#buildartifactpaths) field in **eas.json**.
```